oval-schemas/independent-system-characteristics-schema.xsd

```diff
@@ -325,6 +325,64 @@
+          </xsd:annotation>
+          <xsd:complexType>
+               <xsd:complexContent>
+                    <xsd:extension base="oval-sc:ItemType">
+                         <xsd:sequence>
+                              <xsd:element name="shell" type="oval-sc:EntityItemStringType" minOccurs="1" maxOccurs="1">
+                                   <xsd:annotation>
+                                        <xsd:documentation>The shell element contains the shell used (e.g. bash or powershell) to perform the command and should be taken, verbatim, from the associated object 'shell' element.</xsd:documentation>
+                                   </xsd:annotation>
+                              </xsd:element>
+                              <xsd:element name="command" type="oval-sc::EntityItemStringType"  minOccurs="1" maxOccurs="1">
+                                   <xsd:annotation>
+                                        <xsd:documentation>The command element specifies the command string that was run on the target system and should be taken, verbatim, from the associated object 'command' element..</xsd:documentation>
+                                   </xsd:annotation>
+                              </xsd:element>                              
+                              
+                              <xsd:element name="pattern" type="oval-sc:EntityItemStringType" minOccurs="0" maxOccurs="1">
+                                   <xsd:annotation>
+                                        <xsd:documentation>The pattern element is simply an echo of the same element in the OVAL object and is supplied in the item to aid in end user interpretation and should be taken, verbatim, from the associated object 'pattern' element..</xsd:documentation>
+                                   </xsd:annotation>
+                              </xsd:element>                              
+							<xsd:element name="exit_status" type="oval-sc:EntityItemIntType" minOccurs="1" maxOccurs="1">
+								<xsd:annotation>
+									<xsd:documentation>The exit_status entity represents the exist status returned by the system for the execution of the object command. OVAL Item status should match the exit status of the system call.</xsd:documentation>
+								</xsd:annotation>
+							</xsd:element>
+							<xsd:element name="stdout_line" type="oval-sc:EntityItemStringType" minOccurs="0" maxOccurs="unbounded">
+								<xsd:annotation>
+									<xsd:documentation>The stdout_line entity contains a line from the STDOUT output of a successful run of the command string that matched the specified object pattern.  Each line created by the execution of the object command should create an item 'stdout_line' element.</xsd:documentation>
+								</xsd:annotation>
+							</xsd:element>
+                            <xsd:element name="subexpression" type="oval-sc:EntityItemAnySimpleType" minOccurs="0" maxOccurs="unbounded">
```

Comment:
I'd just make it EntityItemStringType.  The state comparison would govern the type, if needed.

Reply:
Current design matches textfilecontent's subexpression design.  Should we change both this and textfilecontent, or leave them both as-is?

Reply:
@vanderpol No kidding! Well, there's certainly no harm in its being an EntityItemAnySimpleType, so might as well just leave it.